src/auto-calibrated-device.h

```diff
@@ -29,6 +30,7 @@ namespace librealsense
         virtual void set_calibration_config(const rs2_calibration_config& calib_config) = 0;
         virtual std::string calibration_config_to_json_string(const rs2_calibration_config& calib_config) const = 0;
         virtual rs2_calibration_config json_string_to_calibration_config(const std::string& json_str) const = 0;
+        virtual void set_device_for_auto_calib(debug_interface* device) = 0;
```

Comment:
Not for this PR, but I think that the whole `auto_calibrated_interface` should be refactored.
It is very D400 oriented, and even there not all functions are suitable to all camera models.

Reply:
ok - great enablement - could you open a ticket please?